src/common/response_service/filter/all_exceptions.filter.ts

```diff
@@ -16,38 +14,6 @@ export class AllExceptionsFilter extends BaseExceptionFilter {
     const ctx = host.switchToHttp();
```

Comment:
Let's rename this to context? or similar for major clarity please

Reply:
Fixed